lib/arena.rb

```diff
@@ -1,2 +1,66 @@
+# require "pry"
+
+class Gladiator
+
+  attr_accessor :name, :weapon
+
+  def initialize(name, weapon)
+
+    @name = name
+    @weapon = weapon
+
+  end
+end
+
+
 class Arena
+
+
+  attr_accessor :name , :gladiators
+
+
+
+  def initialize(name)
+
+    @name = name.capitalize
+    @gladiators = []
+
+  end
+
+  def add_gladiator(gladiator)
+    if gladiators.length < 2
+      gladiators << gladiator
+    end
+    if gladiators.count == 2
+    end
+  end
+
+  def fight()
+
+    if gladiators.length < 2
+      puts "Winner"
+      return
```

Comment:
Think about when do we want want to "remove a gladiator in an arena":
 - only when there are 2 gladiators in the pit
how are you comparing their weapons to determine winner?